runtime/battery-station/src/parameters.rs

```diff
@@ -144,7 +144,6 @@ parameter_types! {
 
     // ORML
     pub const GetNativeCurrencyId: CurrencyId = Asset::Ztg;
-    pub DustAccount: AccountId = PalletId(*b"orml/dst").into_account_truncating();
```

Comment:
Should we look up the amount of funds on this account? Who controls this account?

Reply:
Only root can control this account. Lookup is not necessary, since tokens with value were not used yet.